compile/functions/runtimes/php.js

```diff
@@ -10,9 +11,19 @@ class Php extends BaseRuntime {
   }
 
   processActionPackage (handlerFile, zip) {
-    return zip.file(handlerFile).async('nodebuffer').then(data => {
-      zip.remove(handlerFile)
-      return zip.file('index.php', data)
+    var fileName = handlerFile, pathSeparatorIndex = handlerFile.lastIndexOf('/');
```

Comment:
Can you change the use of `var` to `let` and `const` for those two variables?
fileName changes but pathSeparatorIndex does not.